setup.cfg

```diff
@@ -0,0 +1,18 @@
+[options]
+package_dir =
+    = src
+packages = find:
+include_package_data = True
+
+[options.packages.find]
+where = src
+
+[flake8]
+# See https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length
+max-line-length = 88
+extend-ignore = E203
+exclude = .*,__pycache__,build,dist,venv
+
+[yapf]
```

Comment:
I think you are using `black` (in the `pre-commit` hooks?).
So you probably don't need a `yapf` section?